source/_patterns/04-pages/home.mustache

```diff
@@ -27,17 +27,22 @@
        </div>
        {{/highlight}}
 
-      <div class="wrapper wrapper--listing">
+      <div class="wrapper wrapper--listing wrapper--listing-selector">
```

Comment:
I don't think there is any need to introduce the wrapper--listing-selector class here. It's not needed for anything. The view-selector has all the info in the dom it needs.

Reply:
The class "wrapper--listing-selector" is not needed and I will remove it but "wrapper--listing" is adding aditional padding-top to the component and if we the js is disabled on the browser, we will hide the tabs and there won't be enough space between the sections, so for me it is better to reuse that class